email-templates/user-lockout-en_US/user-lockout.html.ejs

```diff
@@ -0,0 +1,15 @@
+<% include ../common-en_US/header.ejs %>
+
+<h1>Hi <%=name%>,</h1>
+
+<p>We have noticed multiple failed attempts to login to your zen account so for security reasons have locked your account.  </p>
```

Comment:
Split into 2 sentences to remove the awkward "so".
"We have noticed multiple failed attempts to login to your zen account" "For security reasons, we locked your account".